scripts/get-token.js

```diff
@@ -1,23 +1,33 @@
-import fs from 'fs'
-import dotenv from 'dotenv'
-import FirebaseTokenGetter from 'firebase-idtoken-getter'
```

Comment:
Could we remove this dependency from `package.json`? It's not really being kept up to date.